src/tools/rbd_ggate/ggate_drv.c

```diff
@@ -0,0 +1,375 @@
+// -*- mode:C; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Was this sourced from a different location / project? I ask since it's an interesting separation between ```Driver``` and this plus the debug logging helper functions in "debug.h"